src/ua.c

```diff
@@ -1437,7 +1437,6 @@ static int add_transp_af(const struct sa *laddr)
 				if (err) {
 					warning("ua: tls_add_ca() failed:"
 						" %m\n", err);
-					return err;
```

Comment:
This causes a ua_stop_all()

```
$ baresip/baresip 
baresip v1.0.0 Copyright (C) 2010 - 2020 Alfred E. Heggestad et al.
ua: adding SIP CA: /etc/ssl/certs/ca-certificates2.crt
tls: Can't read CA file: /etc/ssl/certs/ca-certificates2.crt
ua: tls_add_ca() failed: Invalid argument
ua: init failed (Invalid argument)
ua: stop all (forced=1)
```
If we enable sip_cafile by default we are guessing the file for the user. On some platforms like iOS, Android or if the system path changes this will fail. I dont think this should be a hard fail and cause a complete baresip stop all, it should be same as the file is not provided.

For security it makes no change if the file could not be loaded.

Reply:
sip_cafile is disabled by default. Thus ua_init() (which btw should be called uag_init()) will succeed by default.
```
$ grep cafile docs/examples/config 
#sip_cafile		ca.crt
```
But you are right that it does not change the security if we remove the "return err". Its ok for me.